apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/java/gov/cms/bfd/pipeline/bridge/etl/FissTransformerIT.java

```diff
@@ -343,42 +403,80 @@ private static FissClaimChange createFissClaimChange(
         .build();
   }
 
+  /** TestData return class. */
   private static class TestData {
+    /** Beneficiary Id. */
     private static final String BENE_ID = "beneid";
+    /** Beneficiary First Name}. */
     private static final String BENE_FIRST_NAME = "Firstname";
+    /** Beneficiary Last Name. */
     private static final String BENE_LAST_NAME = "Lastname";
+    /** Beneficiary Middle Initial. */
     private static final String BENE_MID_INIT = "M";
+    /** Beneficiary Date of Birth. */
     private static final String BENE_DOB = "2020-01-01";
+    /** Beneficiary Gender. */
     private static final String BENE_GENDER = "1";
+    /** Fi Document Claim Control Number. */
     private static final String FI_DOC_CLM_CNTL_NUM = "dcn12345678";
+    /** HICN Number. */
     private static final String HIC_NO = "hicno123";
+    /** MBI Number. */
     private static final String MBI = "mbimbimbimbi";
+    /** Admitting Diagnosis Code. */
     private static final String ADMTG_DGNS_CD = "admitcd";
+    /** Claim Frequency Code. */
     private static final String CLM_FREQ_CD = "freqCode";
+    /** Claim From Date. */
     private static final String CLM_FROM_DT = "01-Jan-2001";
+
+    /** Claim Type Code returns {@link int}. */
     private static final int CLM_SRVC_CLSFCTN_TYPE_CD = 1;
+    /** Claim Thru Date returns {@link String}. */
     private static final String CLM_THRU_DT = "03-Mar-2001";
+    /** Claim Tot Charge Amount returns {@link String}. */
```

Comment:
Last thing, still some of these remaining.

Reply:
done